CHANGELOG.md

```diff
@@ -101,6 +101,7 @@ New deprecation(s):
 ### Other
 
 - **General**: Replace deprecated webhook.Validator with webhook.CustomValidator ([#6660](https://github.com/kedacore/keda/issues/6660))
+- **General**: Fix several typos ([#6909](https://github.com/kedacore/keda/pull/6909))
```

Comment:
Could you help fix this line? it should be sorted alphabetically like indicated in the failed CI